PyGame/package/Pygame/Pygame.py

```diff
@@ -0,0 +1,17 @@
+
+import pygame
+run = True
+width = 800
+height = 200
+pygame.init()
+screen =pygame.display.set_mode((width, height))
+font = pygame.font.SysFont(None, 55)
+text = font.render("Welcome to Paygame", True, (245, 245, 245))
+screen.blit(text, ((width - text.get_width()) // 2, (height - text.get_height())))
+pygame.display.flip()
+while run:
+    for event in pygame.event.get():
+        if event.type == pygame.quit\
+        or event.type == pygame.MOUSEBUTTONUP\
+        or event.type == pygame.KEYUP:
+            run = False
```

Comment:
if you want to use the package, you should don't use the 'side effect' of your imports.
Better to use the next statement:

```suggestion
def Welcome():
    run = True
    width = 800
    height = 200
    pygame.init()
    screen =pygame.display.set_mode((width, height))
    font = pygame.font.SysFont(None, 55)
    text = font.render("Welcome to Paygame", True, (245, 245, 245))
    screen.blit(text, ((width - text.get_width()) // 2, (height - text.get_height())))
    pygame.display.flip()
    while run:
        for event in pygame.event.get():
            if event.type == pygame.quit\
            or event.type == pygame.MOUSEBUTTONUP\
            or event.type == pygame.KEYUP:
                run = False
```


and after these changes in your main.py use the Welcome function call